style.css

```diff
@@ -0,0 +1,125 @@
+body {
+  background-image: url(wood.png);
+  background-position: left top;
+  background-repeat: repeat;
+  margin: 0;
+}
+
+.beer-wrapper {
+	position: relative;
+  top: 8rem;
+	margin: auto;
+	max-width: 80%;
+	min-width: 440px;
+}
+
+.float-beer {
+	float: left;
+	padding: 1rem;
+  margin: auto;
+  }
+
+.bottle-icon {
+  width: 300px;
+}
+
+.beer-name {
+	text-align: center;
+}
+
+.price {
+	text-align: center;
+}
+
+.disc {
+	text-align: center;
+  max-width: 300px;
+}
+
+.cart{
+  text-align: center;
+}
+
+.cart > a {
+  padding: 5px;
+  border-radius: 25%;
+  color: white;
+  text-decoration: none;
+  background-color: grey;
+}
+/*.feature-image{
+  width: 23%;
+  padding-bottom: 23%;
+  margin: .5em 1%;
+  display: block;
+  float: left;
+}*/
+/*.header-image{
+  width: 98%;
+  margin: .5em 1%;
+  height: 500px;
+}*/
+
+@media screen and (max-width: 1024px) {
```

Comment:
Look at making this media query have both a `max-width` and a `min-width`.

Since the tablet layout is so different than the grid setup for Phone and desktop, if you make these rules ONLY apply to tablets, then you will need to make fewer tweaks on phone since you won't be overriding everything at the tablet size and then undoing everything to make it go back to a grid on phone. 

Does that make sense?